sympy/stats/stochastic_process_types.py

```diff
@@ -71,6 +74,32 @@ def _set_converter(itr):
         raise TypeError("%s is not an instance of list/tuple/set."%(itr))
     return itr
 
+def _state_converter(itr: tSequence) -> tUnion[Tuple, Range]:
+    """
+    Helper function for converting list/tuple/set/Range/Tuple/FiniteSet
+    to tuple/Range.
+    """
+    if isinstance(itr, (Tuple, set, FiniteSet)):
+        itr = Tuple(*(sympify(i) if isinstance(i, str) else i for i in itr))
+
+    elif isinstance(itr, (list, tuple)):
+        # check if states are unique
+        if len(set(itr)) != len(itr):
+            raise ValueError('The state space must have unique elements.')
+        itr = Tuple(*(sympify(i) if isinstance(i, str) else i for i in itr))
+
+    elif isinstance(itr, Range):
+        # the only ordered set in sympy I know of
+        # try to convert to tuple
+        try:
+            itr = Tuple(*(sympify(i) if isinstance(i, str) else i for i in itr))
+        except ValueError:
+            pass
```

Comment:
if you end up on this line, code will not define the `itr` variable. Will it later raise an exception when calling `return itr`?

Reply:
`itr` is an argument in the function so it is always defined inside the method. We are simply changing it if we can.